package.json

```diff
@@ -3,6 +3,7 @@
     "type": "git",
     "url": "http://github.com/cfpb/cfgov-refresh.git"
   },
+  "license": "CC0-1.0",
```

Comment:
Can we point to our license file instead? We have a lot of exceptions to the CC0 rule.